includes/forms/template.php

```diff
@@ -848,86 +862,104 @@ function give_default_cc_address_fields( $form_id ) {
 			<label for="card_address" class="give-label">
 				<?php esc_html_e( 'Address 1', 'give' ); ?>
 				<?php
-				if ( give_field_is_required( 'card_address', $form_id ) ) { ?>
+				if ( give_field_is_required( 'card_address', $form_id ) ) : ?>
```

Comment:
Can we stick with using brackets? It improves the Scrutinizer score and I'd like to stick with them throughout the project.